articles/application-insights/application-insights-data-model-metric-telemetry.md

```diff
@@ -28,7 +28,7 @@ Metric representing system and process counters:
 | **.NET name**             | **Platform agnostic name** | **REST API name** | **Description**
 | ------------------------- | -------------------------- | ----------------- | ---------------- 
 | `\Processor(_Total)\% Processor Time` | Work in progress... | [processorCpuPercentage](https://dev.applicationinsights.io/apiexplorer/metrics?appId=DEMO_APP&apiKey=DEMO_KEY&metricId=performanceCounters%2FprocessorCpuPercentage) | total machine CPU
-| `\Memory\Available Bytes`                 | Work in progress... | [memoryAvailableBytes](https://dev.applicationinsights.io/apiexplorer/metrics?appId=DEMO_APP&apiKey=DEMO_KEY&metricId=performanceCounters%2FmemoryAvailableBytes) | memory available on disk
+| `\Memory\Available Bytes`                 | Work in progress... | [memoryAvailableBytes](https://dev.applicationinsights.io/apiexplorer/metrics?appId=DEMO_APP&apiKey=DEMO_KEY&metricId=performanceCounters%2FmemoryAvailableBytes) | Shows the amount of physical memory, in bytes, available to processes running on the computer. It is calculated by summing adding the amount of space on the zeroed, free, and standby memory lists. Free memory is ready for use; zeroed memory consists of pages of memory filled with zeros to prevent later processes from seeing data used by a previous process; standby memory is memory that has been removed from a process's working set (its physical memory) en route to disk but is still available to be recalled. See [Memory Object](https://msdn.microsoft.com/en-us/library/ms804008.aspx)
```

Comment:
@SergeyKanzhelev 

_It is calculated by **summing adding** the amount of space on the zeroed, free, and standby memory lists._ 

I am guessing you want only **summing** or **adding** rather than both back to back in that sentence? 

If you modify the pull request to drop one I would be happy to sign-off.

Thanks!